dist/controller/entryController.js

```diff
@@ -27,8 +27,12 @@ var EntryController = function (_ClientController) {
   _createClass(EntryController, [{
     key: 'create',
     value: function create(req, res, next) {
+      var _req$body = req.body,
```

Comment:
Why are you implementing changes in your dist?

Reply:
I didn't change it myself, it is as a result of transpilation by babel. I run the build before pushing.